src/api/location/index.ts

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
+import { NAVER_MAPS_CLIENT_ID, NAVER_MAP_CLOUD_SECRET_ID } from 'config/constants';
+
+const getAddress = async (latitude: number, longitude: number): Promise<string> => {
+  const response = await axios.get('/map-reversegeocode/v2/gc', {
+    params: {
+      coords: `${longitude},${latitude}`,
+      sourcecrs: 'epsg:4326',
+      orders: 'roadaddr',
+      output: 'json',
+    },
+    headers: {
+      'X-NCP-APIGW-API-KEY-ID': NAVER_MAPS_CLIENT_ID,
+      'X-NCP-APIGW-API-KEY': NAVER_MAP_CLOUD_SECRET_ID,
+    },
+  });
+
+  const { data } = response;
+  if (data?.results?.length > 0) {
+    return `${data.results[0].region.area1.name} ${data.results[0].region.area2.name}`;
```

Comment:
요걸로 변경 한 후에 폰트 그기 줄여야할 것 같아요!
```suggestion
    return `${data.results[0].region.area1.name} ${data.results[0].region.area2.name} ${data.results[0].region.area3.name} ${data.results[0].land.name} ${data.results[0].land.number1}`;
    
```

Reply:
변경후에 폰트는 그대로 두고 글자 길이에 따라 너비 조절될수 있게 바꿨습니다! 
64369f280d46b5358df6b980f7cd4d4f3118e01c